packages/bierzo-wallet/src/config/index.ts

```diff
@@ -26,7 +26,7 @@ export interface ChainSpec {
 
 export interface FaucetSpec {
   readonly uri: string;
-  readonly token: string;
+  readonly token: ReadonlyArray<string>;
```

Comment:
please rename to plural `tokens` is you want to support a list here. However, right now one token per faucet should be sufficient